src/main/java/com/zufar/onlinestore/product/service/ProductApi.java

```diff
@@ -0,0 +1,10 @@
+package com.zufar.onlinestore.product.service;
+
+import com.zufar.onlinestore.product.dto.ProductResponseDto;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+
+public interface ProductApi {
+
+    Page<ProductResponseDto> getProducts(Pageable pageable);
```

Comment:
The same. Our API should be clear and not depend on  org.springframework.data.domain.Page and  org.springframework.data.domain.Pageable;